tests/test_admin_login_flow.py

```diff
@@ -76,3 +76,24 @@ def test_non_verified_user_is_logged_out(settings, totp_device, client: Client):
     assert login_response.wsgi_request.path == admin_login_url
     assertTemplateUsed(login_response, "maykin_2fa/login.html")
     assertContains(login_response, "Token")
+
+
+def test_mfa_disabled_respects_next_parameter(settings, client: Client, admin_user):
```

Comment:
Type annotation for `client` but not the other params. Oversight or OK as is?

Reply:
ok as is, `settings` and `admin_user` are pytest-django fixtures